src/Sulu/Component/Util/ArrayUtils.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/*
+ * This file is part of the Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Util;
+
+use Symfony\Component\ExpressionLanguage\ExpressionLanguage;
+
+/**
+ * Collection utilities working with symfony-expressions.
+ */
+final class ArrayUtils
```

Comment:
@webmozart Actually the only reason for me would be to be consistent, since the `SortUtils` class is not final. Would probably be a good idea to do this as well. And private constructor definitely makes sense :-)